src/Model/ConcertManager.php

```diff
@@ -60,8 +60,8 @@ public function selectAllArtists(): array
     {
         $query = "SELECT * FROM artist";
         $statement = $this->pdo->query($query);
-        $allArtists = $statement->fetchAll();
+        $allTables = $statement->fetchAll();
```

Comment:
ok it's getting weird ... Let's say "allTables" is never a good name for a variable, EXECPT if you work at IKEA and you have to present a page with ... yeah all the Tables

Reply:
maybe i gonna be better for return working at Ikea, i hope it's good now